sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,5 @@
+SELECT "countries"."name" as "countries",
```

Comment:
I suggest you leave off the plurality, since each `"name"` refers to one country.